install/helm/supergloo/values-defaults.yaml

```
@@ -16,6 +16,9 @@ supergloo:
       repository: quay.io/solo-io/supergloo
       pullPolicy: Always
     stats: true
+  sidecarInjectorImage:
+    repository: quay.io/solo-io/sidecar-injector
```

what about this? (as in above suggestion)
```suggestion
    imageRepository: quay.io/solo-io
```

---

same as above